### PR TITLE
Allows you to throw hats onto people

### PR DESCRIPTION
**Pull request by redmoogle**
# Document the changes in your pull request

Muh immersion

# Wiki Documentation

33% Chance
Need to target head on help intent

# Changelog

:cl:  
rscadd: You can now throw hats onto people if you target their head and are on help intent
/:cl:

### CODE REVIEW

**Comment by NCRLouTenant** [comment]
Another fun little PR. Love it. 

Catch Aurora passing out hats when this gets merged.

**Comment by tattax** [comment]
you'd probably want tinfoil hats to not be able to land on people's heads

File: code/modules/clothing/head/_head.dm [hunk 1]
```diff
@@ -28,3 +28,16 @@
...
+	. = ..()
+	if(throwingdatum?.thrower.zone_selected != BODY_ZONE_HEAD && throwingdatum?.thrower.a_intent != INTENT_HELP)
+		return
+	if(ishuman(hit_atom))
+		var/mob/living/carbon/human/H = hit_atom
+		if(prob(33) && H.equip_to_slot_if_possible(src, SLOT_HEAD))
+			H.visible_message("The [src] lands gracefully on [H]'s head")
+			return TRUE
+		else
+			H.visible_message("The [src] hits [H]'s head")
```
Comment (tattax) [inline code/modules/clothing/head/_head.dm]:
```suggestion
		H.visible_message("The [src] hits [H]'s head")
```

**Comment by warface1234455** [comment]
Time to place c4 on hat and throw em at people

**Comment by Marmio64** [comment]
Should restrict helmets and stuff from this, I really don't know how you'd land a bulky sec helmet on someone's head or a spacesuit helmet.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**